src/vector.rs

```diff
@@ -151,6 +150,36 @@ impl<T, U> Vector2D<T, U> {
     pub fn from_untyped(p: Vector2D<T, UnknownUnit>) -> Self {
         vec2(p.x, p.y)
     }
+
+    /// Computes the vector with absolute values of each component.
+    ///
+    /// For `f32` and `f64`, `NaN` will be returned for component if the component is `NaN`.
+    ///
+    /// For signed integers, `::MIN` will be returned for component if the component is `::MIN`.
```

Comment:
This isn't quite true, `MIN.abs()` causes a panic on debug builds with in. Please remove the two lines that describe the edge cases here. The behavior of `abs` is the responsibility of whichever scalar type the user will put here.